src/lib/helper/getNumOfSeatsPerNation.ts

```diff
@@ -0,0 +1,8 @@
+export default function getNumOfSeatsPerNation(nation: any, committees: any[]) {
```

Comment:
How do I type this correctly? @m1212e

Reply:
I'd recommend using the `import type { Committee, Nation } from '@prisma/client';` import in combination with `Pick<>` to get the smalles amount of fields necessary for the helper funciton. This leaves the function signature to something like this:

```ts
export default function getNumOfSeatsPerNation(
	nation: Pick<Nation, 'alpha3Code'>,
	committees: (Pick<Committee, 'numOfSeatsPerDelegation'> & {
		nations: Pick<Nation, 'alpha3Code'>[];
	})[]
)
```